File: java/URI_1048.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1048 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double salario = sc.nextDouble();
		double novoSalario;		
		
		if(salario <= 400.0) {						
			novoSalario = salario * 1.15;			
		}
		else if(salario <= 800.0) {			
			novoSalario = salario * 1.12;
		}
		else if(salario <= 1200.0) {			
			novoSalario = salario * 1.10;
		}
		else if(salario <= 2000.0) {			
			novoSalario = salario * 1.07;
		}
		else {
			novoSalario = salario * 1.04;
		}
		
		double reajuste = novoSalario - salario;
		double percentual = (novoSalario / salario - 1) * 100;
		System.out.printf("Novo salario: %.2f%n", novoSalario);
		System.out.printf("Reajuste ganho: %.2f%n", reajuste);
		System.out.printf("Em percentual: %.0f %%%n", percentual);
		sc.close();

	}

}

File: kotlin/URI_1048.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)

    val salario = sc.nextDouble()
    val novoSalario = when (salario) {
        in 0.0..400.00 -> salario * 1.15
        in 400.01..800.00 -> salario * 1.12
        in 800.01..1200.00 -> salario * 1.1
        in 1201.00..2000.00 -> salario * 1.07
        else -> salario * 1.04
    }
    val reajuste = novoSalario - salario
    val percentagem = reajuste / salario * 100
    println("Novo salario: ${"%.2f".format(novoSalario)}")
    println("Reajuste ganho: ${"%.2f".format(reajuste)}")
    println("Em percentual: ${"%.0f".format(percentagem)} %")
}
